modules/compute/virtual-machine-scale-set/.test/linux/main.test.bicep

```diff
@@ -9,7 +9,7 @@ targetScope = 'subscription'
 param resourceGroupName string = 'ms.compute.virtualmachinescalesets-${serviceShort}-rg'
 
 @description('Optional. The location to deploy resources to.')
-param location string = deployment().location
+param location string = 'westeurope' //deployment().location
```

Comment:
Was that intentionally?

Reply:
no only a test... I missed to restore to deployment().location